s3dol/tests/test_base.py

```diff
@@ -0,0 +1,160 @@
+import os
+from s3dol.base import S3BucketDol, S3ClientReader, S3Dol, S3ClientDol, S3DolReadOnly
+import pytest
+
+
+@pytest.mark.parametrize(
+    'aws_access_key_id, aws_secret_access_key, endpoint_url',
+    [('localstack', 'localstack', 'http://localhost:4566')],
+)
+def test_s3_dol_crud(aws_access_key_id, aws_secret_access_key, endpoint_url):
+    os.environ['AWS_ACCESS_KEY_ID'] = aws_access_key_id
```

Comment:
This is overwriting existing env var of the instance running the test with values passed into the function to prevent the use of env vars set by the shell.  This also demonstrates setting env var to instantiate without arguments

Reply:
I don't understand. 

@valentin-feron  -- can you see what @andeaseme means here? (@andeaseme; remind him if he doesn't bring it up.)

Reply:
@valentin-feron  explained what he thinks you mean, and if so, we both disagree. 
Discuss in today's 1-on-1.